shared.c

```diff
@@ -496,6 +498,24 @@ void DeploySober(string p="", string v=""){
 	trUnitTeleport(trVectorQuestVarGetX(v),trVectorQuestVarGetY(v),trVectorQuestVarGetZ(v));
 }
 
+void ChatLogShow(int p = 1){
+	if (trCurrentPlayer() == p) {
+		trChatHistoryClear();
+	}
+	int x = peekModularCounterNext("chat" + p + "Log");
+	for(i=x;<11){
```

Comment:
just use a single for loop and replace i with modularCounterNext() lol. 
`for(i=11; >0) {
trChatSendToPlayer(0, p, trStringQuestVarGet("chat" + p + "log" + modularCounterNext("chat" + p + "log"));
}`

Reply:
Doesn't actually matter since the runtime is the same but w/e